whats-new-in-pgroll-0-10-0.mdx

```diff
@@ -0,0 +1,465 @@
+    ]
+  }
+}
+```
+
+Also, the same applies to some cases when the ORM does the backfilling for new columns. For example, when you add a new column with a default value, an ORM produces these SQL statements:
+
+```sql
+ALTER TABLE "adults" ADD COLUMN "age" smallint DEFAULT 18 NULL CHECK ("age" >= 18);
+ALTER TABLE "adults" ALTER COLUMN "level" DROP DEFAULT;
+
+UPDATE adults SET age = 18;
+
+ALTER TABLE adults ALTER COLUMN age SET NOT NULL;
+
+UPDATE "adults" SET "age" = 18 WHERE "age" IS NULL;
+SET CONSTRAINTS ALL IMMEDIATE;
+
+ALTER TABLE "adults" ALTER COLUMN "age" SET NOT NULL;
+```
+
+Resulting in the following pgroll migration using the `convert` subcommand:
+
+```json
+{
+  "add_column": {
+    "column": {
+      "check": {
+        "constraint": "age \u003e= 0",
+        "name": "age_check"
+      },
+      "default": "18",
+      "name": "age",
+      "nullable": true,
+      "type": "smallint"
+    },
+    "table": "adults",
+    "up": "TODO: Implement SQL data migration"
+  }
+},
+{
+  "alter_column": {
+    "column": "age",
+    "default": null,
+    "table": "adults",
+    "down": "TODO: Implement SQL data migration",
+    "up": "TODO: Implement SQL data migration"
+  }
+},
+{
+  "sql": {
+    "up": "update adults set age = 18"
+  }
+},
+{
+  "alter_column": {
+    "column": "age",
+    "nullable": false,
+    "table": "adults",
+    "up": "TODO: Implement SQL data migration"
+    "down": "TODO: Implement SQL data migration",
+  }
+},
+{
+  "sql": {
+    "up": "UPDATE \"adults\" SET \"age\" = 18 WHERE \"age\" IS NULL"
+  }
+},
+{
+  "sql": {
+    "up": "SET CONSTRAINTS ALL IMMEDIATE"
+  }
+},
+{
+  "alter_column": {
+    "column": "age",
+    "nullable": false,
+    "table": "adults",
+    "up": "TODO: Implement SQL data migration"
+    "down": "TODO: Implement SQL data migration",
+  }
+}
+```
+
+This can be written as a single pgroll migration:
+
+```json
+{
+  "add_column": {
+    "column": {
+      "check": {
+        "constraint": "age \u003e= 0",
+        "name": "age_check"
+      },
+      "default": "18",
+      "name": "age",
+      "nullable": true,
+      "type": "smallint"
+    },
+    "table": "adults",
+    "up": "18"
+  }
+},
+```
+
+Also, SQL migration files usually start with `BEGIN` and `COMMIT` because the ORM runs the DDLs in a single transaction. These staments show up in the list of operations in the generated migrations. These operations are safe to be deleted from the list.
+
+## Breaking change to create_index operations
+
+`pgroll` `v0.10` includes one breaking change to how columns are configured when creating new indexes. Previously, the `columns` list of `create_index` operations expected a list of strings:
+
+```json
+{
+  "name": "10_create_index",
+  "operations": [
+    {
+      "create_index": {
+        "table": "employees",
+        "columns": ["salary"]
+      }
+    }
+  ]
+}
+```
+
+Now `create_index` supports several new options to configure the index elements per column.
+
+### Sort order
```

Comment:
The most common case is to create the index on a column or columns with no options, so I think we should have an example of that too.